.ci/ci.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
```

Comment:
It is recommended to have `set -e` on a separate line so that the scripts exits on fail even if it is called like so: `bash ci.sh`.

IMO names of shell scripts should also not be suffixed with `.sh`. The fact that the application is written as a shell script is an implementation detail.

I typically use [google style guide](https://google.github.io/styleguide/shell.xml) for shell scripts.

Reply:
:+1: 
Done

Reply:
> bash ci.sh.

...

> an implementation detail.

:laughing: